install_config/topics/configuring_azure_authorization.adoc

```diff
@@ -0,0 +1,77 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-permissions_{context}']
+= Configuring authorization for Microsoft Azure
+
+*Azure roles*
+
+Configuring Microsoft Azure for {product-title} requires the following role:
+
+[cols='1,3']
+|===
+
+|Contributor
+|To create and manage all types of Microsoft Azure resources.
+
+|===
+
+See the
+link:https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles[Classic
+subscription administrator roles vs. Azure RBAC roles vs. Azure AD administrator
+roles documentation] for more information.
+
+*Permissions*
+
+Configuring Microsoft Azure for {product-title} requires a service principal,
+which allows the creation and management of Kubernetes service
+load balancers and disks for persistent storage. The service principal values
+are defined at installation time and deployed to the Azure configuration file, located at `/etc/origin/cloudprovider/azure.conf` on {product-title} master and node hosts.
+
+.Procedure
+
+. Using the Azure CLI, obtain the account subscription ID:
++
+[subs=+quotes]
+----
+# az account list
+[
+{
+  "cloudName": "AzureCloud",
+  "id": "<subscription>", <1>
+  "isDefault": false,
+  "name": "Pay-As-You-Go",
+  "state": "Enabled",
+  "tenantId": "<tenant-id>",
+  "user": {
+    "name": "admin@example.com",
+    "type": "user"
+  }
+]
+----
+<1> The subscription ID to use to create the new permissions.
+
+. Create the service principal with the
+role of contributor and with the scope of the Microsoft Azure subscription and
+the resource group. Record the output of these values to be used when defining
+the inventory. Use the `<subscription>` value from the previous step in place of the value below:
++
+[subs=+quotes]
+----
+# az ad sp create-for-rbac --name openshiftcloudprovider \
+     --password <secret> --role contributor \
+     --scopes /subscriptions/<subscription>/resourceGroups/<resource-group>
+
+Retrying role assignment creation: 1/36
+Retrying role assignment creation: 2/36
+{
+  "appId": "<app-id>",
+  "displayName": "ocpcloudprovider",
+  "name": "http://ocpcloudprovider",
+  "password": "<secret>",
+  "tenant": "<tenant-id>"
+}
+----
```

Comment:
@bfallonf Just reading this topic, it seems confusing as to where the configurations are made, Azure or OpenShift. 

It leaves me wondering if we shouldn't add subtopics: Configuring Azure; Configuring {product-title}